extension/js/common/composer.ts

```diff
@@ -210,8 +205,9 @@ export class Composer {
     this.S.cached('input_text').get(0).onpaste = this.inputTextPasteHtmlAsText;
     this.S.cached('icon_footer').click(Ui.event.handle(target => {
```

Comment:
There probably doesn't need to be a footer icon anymore, either. User can change their footer on Gmail, or edit it by clicking the three dots.